### PR TITLE
Remove gitter

### PR DESCRIPTION
**Pull request by codyloyd**
this pr removes mentions of gitter all around the site and replaces them with discord.. I'm fairly certain that I got all the links and buttons

there are a couple of analytics things and related css classes that I left, mostly because it doesn't matter and I didn't want to break anything.

I pushed all the changes to https://staging-odin-project.herokuapp.com/ if you wanna SEE it.

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -122,7 +122,7 @@ def faq_items
       {
         question: 'How can I get in touch?',
         answer:
-        "<p>If you have questions about The Odin Project itself, feel free to <a href='mailto:contact@theodinproject.com'>contact us</a>.  There is also a friendly Odin community in our <a href='https://gitter.im/TheOdinProject/theodinproject'>Gitter chat rooms</a></p>"
+        "<p>If you have questions about The Odin Project itself, feel free to <a href='mailto:contact@theodinproject.com'>contact us</a>.  There is also a friendly Odin community in our <a href='https://discord.gg/hvqVr6d'>Discord chat rooms</a></p>"
```
Comment (arun1595) [inline app/helpers/application_helper.rb]: What do you think of using the `chat_link` method instead of hardcoding the URL?

Comment (codyloyd) [reply app/helpers/application_helper.rb]: no reason not to.... good catch

Comment (codyloyd) [reply app/helpers/application_helper.rb]: well maybe there's a reason not to

Comment (codyloyd) [reply app/helpers/application_helper.rb]: looking at that file.. those are just strings that get turned into html at some point.. I don't think I can do a `link_to` in there... and this doesn't really hurt anything.

**Pull request merged**